config.js.template

```diff
@@ -1,10 +1,10 @@
 //globals: equal, responseText, statement, ok, deepEqual, QUnit, module, asyncTest, Util, start, golfStatements, console
 /*jslint bitwise: true, browser: true, plusplus: true, maxerr: 50, indent: 4 */
 function Config() {
-	"use strict";
+    "use strict";
 }
 Config.endpoint = "<TCAPI end point (ending with slash)>";
 Config.authUser = "<account id>";
 Config.authPassword = "<password>";
-Config.actor = { "mbox":["<learner email>"], "name":["<Learner Name>"] };
+Config.actor = { "mbox":"<learner email>", "name":"<Learner Name>" };
```

Comment:
Did the code that uses Config.actor all get reviewed to make sure it isn't expecting an array still? I seem to recall us leaving this in place because the launcher code expected it to look like an array even though post-0.9 an actual Agent doesn't use an array. I'm okay with making this change, so long as the code has been fixed and tested.